assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -670,9 +673,12 @@ export default Component.extend({
   },
 
   @action
-  editMessage(chatMessage, newContent) {
+  editMessage(chatMessage, newContent, uploads) {
     this.set("sendingloading", true);
-    let data = { new_message: newContent };
+    let data = {
+      newMessage: newContent,
```

Comment:
We should be using `snake_case` for parameters to ruby controllers, so `new_message` and `upload_ids` etc.

Reply:
Okay, I was avoiding this so I could reuse the same JS object for settings attrs, but I'll do that :)